docs/content/Hidden.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Hidden
+description: Use Hidden to responsively hide or show content in narrow, regular and wide viewports.
+source: https://github.com/primer/react/blob/main/src/Hidden/index.tsx
+status: Alpha
+componentId: hidden
+---
+
+The `Hidden` component is a `<span />` that will help you hide or show content/components in different viewports.
+**Attention:** Use Hidden only to control behaviour in a responsive manner. It does not take any `sx` props.
```

Comment:
Quick question, would there be a way to control block vs inline flow for this component? Or is the intention that it should always be inline?

Reply:
Does hidden have to render any DOM element? Could it be something like this:

```jsx
function Hidden({when, children}) {
  const isHidden = useResponsiveValue(...)
  return !isHidden ? children : null
}

Reply:
yeah so the tossup was between do a `display:none` or not render the component at all. I like not rendering. reduces markup + no need to specify `display` value at all.